src/components/MovieCards/MovieCards.js

```diff
@@ -1,7 +1,18 @@
-function MovieCards({ filteredData }) {
+function MovieCards({ filteredData, setSelectedMovie }) {
+
+  function getDetails(event) {
+    // event.preventDefault();
+    setSelectedMovie(event.target.id)
```

Comment:
See my note in App.js about passing the setter functions vs handler functions that invoke the setters.  Something to consider - ultimately it is your decision (and a great thing to solicit input and opinions from some alumni on!)